src/components/mixins/ProfilesMixin.vue

```diff
@@ -33,5 +33,10 @@ export default class ProfilesMixin extends Vue {
         return sanitize(nameToSanitize);
     }
 
+    isProfileCodeValid(profileImportCode: string): boolean {
+        const REGEX = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i;
```

Comment:
Defining the regexp inside the function means an object gets created each time the function is called. Given that it's relatively simple case and probably won't be called that many times(*), the overhead caused by this might be negligible. However, it would be better to define it on the "root" level of the module, i.e. before this class definition.

(*): it's hard to estimate how many times Vue calls this though. Might be a lot?